validation/shared/src/main/scala/hmda/validation/dsl/CommonDsl.scala

```diff
@@ -0,0 +1,76 @@
+package hmda.validation.dsl
+
+trait CommonDsl {
+  implicit class Rule[T](data: T) {
```

Comment:
@schbetsy had (and I like) the idea of calling this class Subject. It looks like it's not actually a Rule (though I can totally see how it might have started out with that intent).